.gitignore

```diff
@@ -0,0 +1,3 @@
+target
```

Comment:
i thought we said having target and target/ was redundant? why not take one of them out?